py3status/modules/deadbeef.py

```diff
@@ -0,0 +1,88 @@
+# -*- coding: utf-8 -*-
+"""
+Display track currently playing in deadbeef.
+
+Configuration parameters:
+    - cache_timeout : how often we refresh usage in seconds (default: 1s)
+    - format : see placeholders below
+    - delimiter : delimiter character for parsing (default: ¥)
+
+Format of status string placeholders:
+    {artist} : artist
+    {title} : title
+    {elapsed} : elapsed time
+    {length} : total length
+    {year} : year
+    {tracknum} : track number
+
+@author mrt-prodz
+"""
+from subprocess import check_output
+from time import time
+
+
+class Py3status:
+    # available configuration parameters
+    cache_timeout = 1
+    delimiter = '¥'
+    format = '{artist} - {title}'
+
+    # return error occurs
+    def _error_response(self, color):
+        response = {
+            'cached_until': time() + self.cache_timeout,
+            'full_text': 'deadbeef: error',
+            'color': color
+        }
+        return response
+
+    # return track currently playing in deadbeef
+    def get_status(self, i3s_output_list, i3s_config):
```

Comment:
It should not start Deadbeef if it is not already running. Deadbeef start up is very expensive:
```
$ time deadbeef --nowplaying-tf "%artist% %title%" &> /dev/null

real	0m2.258s
user	0m2.093s
sys	0m0.097s
```